api/README.md

```diff
@@ -0,0 +1 @@
+# api7-manager-api
```

Comment:
Please add more info about the manager API.

Reply:
added a little description